indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -214,6 +218,13 @@ public TaskStatus call()
                         command.add("-cp");
                         command.add(taskClasspath);
 
+                        if (numProcessorsPerTask < 1) {
```

Comment:
It wouldn't be needed if start and run are called in the proper order. It's here as a safety check in case they aren't. (Maybe it would help debug tests that are doing things incorrectly.)